lucene/misc/native/src/main/posix/NativePosixUtil.cpp

```diff
@@ -38,12 +38,12 @@
 
 #ifdef LINUX
 /*
- * Class:     org_apache_lucene_store_NativePosixUtil
+ * Class:     org_apache_lucene_misc_store_NativePosixUtil
  * Method:    posix_fadvise
  * Signature: (Ljava/io/FileDescriptor;JJI)V
  */
 extern "C"
-JNIEXPORT jint JNICALL Java_org_apache_lucene_store_NativePosixUtil_posix_1fadvise(JNIEnv *env, jclass _ignore, jobject fileDescriptor, jlong offset, jlong len, jint advice)
+JNIEXPORT jint JNICALL Java_org_apache_lucene_misc_store_NativePosixUtil_posix_1fadvise(JNIEnv *env, jclass _ignore, jobject fileDescriptor, jlong offset, jlong len, jint advice)
```

Comment:
These paths need to be adjusted as the native code has been moved in d1110394e9c

Reply:
Yeah. We really should try to add a test that tries to run with these libs. I don't know how to handle this yet - can be a follow-up issue.